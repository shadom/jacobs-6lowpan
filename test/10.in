1.3.6.1.2.1.1.11.0 1.4

