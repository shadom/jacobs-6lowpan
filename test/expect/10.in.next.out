SNMPv2-MIB::system.13.0 = Timeticks: (4290535758) 496 days, 14:09:17.58
n: (noSuchName) There is no such variable name in this MIB.
Failed object: iso.4

